buf/registry/module/v1beta1/branch_service.proto

```diff
@@ -86,6 +87,10 @@ message ListBranchesRequest {
   //   - Is a Branch is referenced, this Branch is returned.
   //   - If a Digest is referenced, all Branches that contain a Commit with this Digest are returned.
   ResourceRef resource_ref = 3 [(buf.validate.field).required = true];
+  // The order to sort the Branches.
+  //
+  // If empty, the default ORDER_ASCENDING is used.
+  buf.registry.v1beta1.Order order = 4 [(buf.validate.field).enum.defined_only = true];
```

Comment:
Just make all of these `bool sort_desc` - there's only two orders for sorting, there isn' the possibility of a third, we want to default to ascending, might as well just make it a `bool`, and if for some reason this doesn't work out, we can later deprecate this field